api/contract.go

```diff
@@ -68,18 +68,6 @@ type putFilterResponse struct {
 	model.PutFilter
 }
 
-// createFilterOutputResponse is the response body for POST /filters-output
-type createFilterOutputResponse struct {
```

Comment:
Removed these as there is no JSON  response for filter output. Just status code is returned